app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidget.kt

```diff
@@ -125,9 +125,16 @@ class StaticWidget : AppWidgetProvider() {
 
         if (attributeIds == null) return entity?.state
 
-        val fetchedAttributes = entity?.attributes as Map<*, *>
-        val attributeValues = attributeIds.split(",").map { id -> fetchedAttributes.get(id)?.toString() }
-        return entity.state.plus(if (attributeValues.isNotEmpty()) stateSeparator else "").plus(attributeValues.joinToString(attributeSeparator))
+        var fetchedAttributes: Map<*, *>
+        var attributeValues: List<String?>
+        try {
+            fetchedAttributes = entity?.attributes as Map<*, *>
```

Comment:
By putting this in a try catch we stop the crash but don't fix the problem.  The problem is `entity?.attributes` is returning null and trying to be cast to a map.  If you add `as?` it will still return null
```suggestion
            fetchedAttributes = entity?.attributes as? Map<*, *>
```

Reply:
Looks like Kotlin wanted extra null checks here :)

```
(entity?.attributes as? Map<*, *>)!!
```